codersdk/idpsync.go

```diff
@@ -163,3 +163,31 @@ func (c *Client) GetOrganizationAvailableIDPSyncFields(ctx context.Context, orgI
 	var resp []string
 	return resp, json.NewDecoder(res.Body).Decode(&resp)
 }
+
+func (c *Client) GetIDPSyncFieldValues(ctx context.Context, claimField string) ([]string, error) {
+	res, err := c.Request(ctx, http.MethodGet, fmt.Sprintf("/api/v2/settings/idpsync/field-values?claimField=%s", claimField), nil)
```

Comment:
`claimField=%s` will not url encode special characters.

```suggestion
	qv := url.Values{}
	qv.Add("claimField", claimField)
	res, err := c.Request(ctx, http.MethodGet, fmt.Sprintf("/api/v2/settings/idpsync/field-values?%s", qv.Encode()), nil)
```

Example: https://go.dev/play/p/zN-CJsgWkz0